lib/handlebars/compiler/javascript-compiler.js

```diff
@@ -1072,6 +1072,7 @@ JavaScriptCompiler.prototype = {
 
   setupHelperArgs: function(helper, paramSize, params, useRegister) {
     let options = this.setupParams(helper, paramSize, params);
+    options.loc = JSON.stringify(this.source.currentLocation);
```

Comment:
This is so I can be able to access the source location from inside helpers it's useful when I want to throw an exception including the location from inside my helper